### DISCUSSION TITLE
DB2 support

### DISCUSSION THREAD

**Discussion by Michuu93** [Keyfactor/ejbca-ce#192] [Q&A]
Can I add DB2 driver (db2jcc4.jar) to container and connect to DB2 database?

**Comment by primetomas** [comment]
I don't think it's that easy. There is some internal/hibernate configuration, like Hibernate dialect that gets configured when changing database.
In the documentation there are some out-of-the-box databases supported. https://hub.docker.com/r/keyfactor/ejbca-ce, by setting the DATABASE_JDBC_URL. I think the startup script that runs when you start the container does some magic depending on what database you select. You have to check what this does and likely modify it... (not sure how many databases it contains).

**Comment by primetomas** [comment]
I think this question should be in Discussions btw. It's not an issue as such.
